Gemfile

```diff
@@ -31,6 +31,7 @@ group :development, :test do
   gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
   gem 'capybara'
   gem 'factory_bot_rails'
+  gem 'rails-controller-testing'
```

Comment:
is this used anywhere? I can't see any `assigns` or `assert_template`.

Reply:
Right, I've been debugging failed test and I needed `assigns`. Maybe it can stay for debugging.